Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -275,6 +276,75 @@ extension VisualInstruction.Component: Codable {
         case lane
     }
     
+    /**
+     :nodoc:
+     Subtype that provides more context about the component guidance view that may help in visual
+     markup and display choices.
+     */
+    public enum SubType: String, Codable, CaseIterable {
+        
+        /**
+         Junction view. Bird’s-eye artist’s rendition view of the overhead signage and
+         preferred road lane arrow on motorways where the road bifurcates into 2 or
+         more motorway trunk roads.
+         */
+        case jct = "jct"
```

Comment:
Swift symbol names should [avoid abbreviation](https://www.swift.org/documentation/api-design-guidelines/#avoid-abbreviations), so this would be `junction`. However, from the documentation, it sounds like this is actually about a fork in the road, not any junction. In that case, we could call it `fork` for consistency with `ManeuverType.reachFork`.